packages/features/bookings/Booker/utils/event.ts

```diff
@@ -108,6 +110,7 @@ export const useScheduleForEvent = ({
     isTeamEvent: pathname?.indexOf("/team/") !== -1 || isTeam,
     orgSlug,
     teamMemberEmail,
+    bookerEmail,
```

Comment:
This was recently removed because it was calling getSchedules endpoint at every key stroke unnecessarily. I am not sure if you added this intentionally or was some merge issue, please remove this.


https://github.com/user-attachments/assets/c8cd2efc-8bd5-453f-af92-550469a8c56a